### PR TITLE
Insert NuGet Build 5.0.0-preview3.5781 into cli

### PR DESCRIPTION
**Pull request by nugetlurker**
Insert NuGet Build 5.0.0-preview3.5781 into cli release/2.2.2xx branch

### CODE REVIEW

**Comment by wli3** [comment]
@dotnet-bot test RHEL6 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test opensuse.42.3 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test Linux x64 Release Build
@dotnet-bot test Linux_NoSuffix x64 Release Build 
@dotnet-bot test RHEL7.2 x64 Release Build 
@dotnet-bot test Ubuntu16.04 x64 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test Linux_NoSuffix x64 Release Build

File: build/DependencyVersions.props [hunk 1]
```diff
@@ -40,10 +40,10 @@
     <MicrosoftDotNetPlatformAbstractionsPackageVersion>2.1.0</MicrosoftDotNetPlatformAbstractionsPackageVersion>
     <MicrosoftExtensionsDependencyModelPackageVersion>2.1.0</MicrosoftExtensionsDependencyModelPackageVersion>
     <MicrosoftDotNetCliCommandLinePackageVersion>0.1.1</MicrosoftDotNetCliCommandLinePackageVersion>
-    <MicrosoftDotNetProjectJsonMigrationPackageVersion>1.3.1</MicrosoftDotNetProjectJsonMigrationPackageVersion>
+    <MicrosoftDotNetProjectJsonMigrationPackageVersion>1.3.1-preview.19067.1</MicrosoftDotNetProjectJsonMigrationPackageVersion>
```
Comment (nguerrera) [inline build/DependencyVersions.props]: This probably should have been bumped to 1.3.2. This version looks older. It doesn't really matter though what the version of this is I guess.

Comment (nguerrera) [reply build/DependencyVersions.props]: Actually at some point a stabilized build will get kicked off again potentially and become 1.3.1 duplicate. We should bump it, but that can happen after this I think.

Comment (wli3) [reply build/DependencyVersions.props]: @nguerrera  good point. Could you approve this branding update https://github.com/dotnet/cli-migrate/pull/109

Comment (nguerrera) [reply build/DependencyVersions.props]: Done. Thanks.

**Comment by wli3** [comment]
@dotnet-bot test Windows_NT x64 Release Build 
@dotnet-bot test Windows_NT_ES x64 Debug Build
@dotnet-bot test Windows_NT_NoSuffix x64 Release Build

**Comment by wli3** [comment]
@dotnet-bot test Windows_NT x86 Debug Build

**Comment by wli3** [comment]
@dotnet-bot test CentOS7.1 x64 Debug Build

**Comment by wli3** [comment]
This test get much more flaky. I need to also investigate that

```
 String "C:\Users\wul\AppData\Local\Temp\2g1yhzmw.jdy\restore.csproj : error NU1101: Unable to find package nonexistent_tool_package. No packages exist with this id in source(s): api.nuget.org, aspnetcore-dev, aspnetcore-release, BlobFeed, BlobFeed-2.1.3-runtime, C:\work\cli\bin\2\win-x64\dotnet\sdk\NuGetFallbackFolder, container-tools, dotnet-core, dotnet-msbuild, linux-musl-bootstrap-feed, myget-vstest, nuget-build, roslyn, templating, websdk (TaskId:57)
                       Restore failed in 2.1 sec for C:\Users\wul\AppData\Local\Temp\2g1yhzmw.jdy\restore.csproj." does not contain visually same fragment string "Restoring". (TaskId:57)
                     Stack Trace: (TaskId:57)
                        at FluentAssertions.Execution.XUnit2TestFramework.Throw(String message) in C:\projects\fluentassertions-vf06b\Src\FluentAssertions.Net45\Execution\XUnit2TestFramework.cs:line 32 (TaskId:57)
                        at FluentAssertions.Execution.AssertionScope.FailWith(String message, Object[] args) in C:\projects\fluentassertions-vf06b\Src\Core\Execution\AssertionScope.cs:line 197 (TaskId:57)
                        at Microsoft.DotNet.Tools.Test.Utilities.StringAssertionsExtensions.ContainVisuallySameFragment(StringAssertions assertions, String expected, String because, Object[] becauseArgs) in C:\work\cli\test\Microsoft.DotNet.Tools.Tests.Utilities\Assertions\StringAssertionsExtensions.cs:line 40 (TaskId:57)
                        at Microsoft.DotNet.Tools.Test.Utilities.StringAssertionsExtensions.ContainVisuallySameFragmentIfNotLocalized(StringAssertions assertions, String expected, String because, Object[] becauseArgs) in C:\work\cli\test\Microsoft.DotNet.Tools.Tests.Utilities\Assertions\StringAssertionsExtensions.cs:line 52 (TaskId:57)
                        at Microsoft.DotNet.Cli.Install.Tests.GivenDotnetInstallTool.ItRunsWithTheSpecifiedVerbosity() in C:\work\cli\test\dotnet-install-tool.Tests\GivenDotnetInstallTool.cs:line 36 (TaskId:57)
```

**Comment by wli3** [comment]
Change test to use "normal" since nuget no longer output restoring... text to minimal

**Comment by wli3** [comment]
@dotnet-bot test Linux-musl x64 Debug Build

**Comment by jainaashish** [comment]
Please abandon this PR.

**Pull request closed without merging**